src/utilities/util.ts

```diff
@@ -580,6 +580,59 @@ export async function requestInitOperatorCollectionInfo(): Promise<string[] | un
   return args;
 }
 
+/**
+ * Prompts the user for the necessary info to create a Credential Secret
+ * @returns - A Promise containing the list of parameters to pass to the command
+ */
+export async function requestCreateCredentialSecreteInfo(): Promise<string[] | undefined> {
+  const args: Array<string> = [];
+  const singleInputRegex = /^\S*$/; // matches single input, no spaces
+
+  const validateSingleInput = (text: string): string | null => {
+    const isValid = text.trim().match(singleInputRegex)?.length;
+    return isValid ? null : "Please enter a single input for this field.";
+  };
+
+  const operatorName = await vscode.window.showInputBox({
+    prompt: 'Enter the operator name. This can be found under the "labels" field in the suboperator pod.',
+    ignoreFocusOut: true,
+    validateInput: text => {
+      return validateSingleInput(text);
+    },
+  });
+
+  const sshKey = await vscode.window.showInputBox({
```

Comment:
Each parameter should perform a validation for `undefined` or an empty string. `undefined` implies that the user clicked escape in which we'd want to return immediately from this function without executing the command. Empty string would imply that the user didn't enter anything in the text field which should be flagged if this field is required.